components/src/dynamo/vllm/handlers.py

```
@@ -367,110 +372,130 @@ async def load_lora(self, request=None):
+                            model_type=ModelType.Chat | ModelType.Completions,
+                            endpoint=self.generate_endpoint,
+                            model_path=self.config.model,
+                            kv_cache_block_size=self.config.engine_args.block_size,
+                            user_data=user_data,
+                            lora_name=lora_name,
+                            base_model_path=self.config.model,
+                        )
+                        logger.info(
+                            f"Successfully published LoRA '{lora_name}' ModelDeploymentCard"
+                        )
+                    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)